tests/Microsoft.Identity.Test.Unit/PublicApiTests/LoggerTests.cs

```diff
@@ -369,6 +370,44 @@ public async Task NullExternalMsalLoggerTestAsync()
 
         }
 
+        [TestMethod]
+        [DataRow(true)]
+        [DataRow(false)]
+        public async Task ClaimsChallengeErrorLogging_TestAsync(bool piiLogging)
```

Comment:
nit: Don't need to add Async to tests anymore.

```suggestion
        public async Task ClaimsChallengeErrorLogging_Test(bool piiLogging)
```